test/e2e-common.sh

```diff
@@ -169,3 +169,12 @@ function delete_pipeline_resources() {
 function delete_resolvers_resources() {
   kubectl delete --ignore-not-found=true resolutionrequests.resolution.tekton.dev --all
 }
+
+function apply_resources() {
+  local resource=$1
+  echo ">> Applying the resource ${resource}"
+  # Applying the resources, either *taskruns or * *pipelineruns
+  for file in $(find ${REPO_ROOT_DIR}/examples/${resource}s/ -name *.yaml | sort); do
+    perl -p -e 's/gcr.io\/christiewilson-catfactory/$ENV{KO_DOCKER_REPO}/g' ${file} | ko apply -f - || return 1
```

Comment:
I added the check for the path that we are iterating before the for loop. 

Particularly I used v1 examples to avoid creating resources that could already exist in the same namespace.